### PR TITLE
[varLib.plot] Add plotModelFromMasters

### PR DESCRIPTION
**Pull request by behdad**

### CODE REVIEW

File: Lib/fontTools/varLib/plot.py [hunk 1]
```diff
@@ -131,6 +131,60 @@ def plotDocument(doc, fig, **kwargs):
...
+	Z = np.array(Z)
+
+	from matplotlib import pyplot
+	fix, ax = pyplot.subplots(subplot_kw={"projection": "3d"})
+
+	surf = ax.plot_surface(X, Y, Z)
+	pyplot.show()
+
+
+def plotModelFromMasters(model, masterValues):
```
Comment (anthrotype) [inline Lib/fontTools/varLib/plot.py:179]: the new function doesn't seem to be used anywhere, do you plan to hook it to main() via some flag I suppose?

Comment (behdad) [reply Lib/fontTools/varLib/plot.py:179]: > the new function doesn't seem to be used anywhere, do you plan to hook it to main() via some flag I suppose?

I can. For now it's a library function for visualization.

Comment (anthrotype) [reply Lib/fontTools/varLib/plot.py:179]: since there's no tests, would be nice to have an example of how to use

Comment (behdad) [reply Lib/fontTools/varLib/plot.py]: Added command-line for it.

**Review by anthrotype — APPROVED** [review]

**Pull request merged**